src/presentational-components/shared/table-composable-toolbar-view.tsx

```diff
@@ -0,0 +1,272 @@
+import React, { Fragment } from 'react';
+import { useIntl } from 'react-intl';
+import messages from '../../Messages';
+import { TableVariant, TableComposable, Thead, Tr, Th, Tbody, Td } from '@patternfly/react-table';
+import TableToolbar from '@redhat-cloud-services/frontend-components/TableToolbar';
+import { Button, Pagination, EmptyStatePrimary, LabelProps } from '@patternfly/react-core';
+import { ListLoader } from './loader-placeholders';
+import { PlusCircleIcon } from '@patternfly/react-icons';
+import Toolbar, { paginationBuilder } from './toolbar';
+import EmptyWithAction from './empty-state';
+import './table-toolbar-view.scss';
+import { ISortBy, OnSort } from '@patternfly/react-table';
+
+// only for users-list-not-selectable for now
+interface RowProps {
+  uuid: string; //username
+  cells: [
+    React.ElementType, //yes or no for isOrgAdmin
+    { title: string | React.RefAttributes<HTMLAnchorElement> }, //link to user or just username
+    string, //email
+    string, //firstName
+    string, //lastName
+    { title: React.FC<LabelProps>; props: { 'data-is-active': boolean } }
+  ];
+  selected: boolean;
+}
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Is this legacy or future type? If the type is not used, we don't want it to be defined.